bitnami/rabbitmq/templates/servicemonitor.yaml

```diff
@@ -27,8 +27,11 @@ spec:
       {{- if .Values.metrics.serviceMonitor.honorLabels }}
       honorLabels: {{ .Values.metrics.serviceMonitor.honorLabels }}
       {{- end }}
-      {{- if .Values.metrics.serviceMonitor.relabellings }}
-      metricRelabelings: {{- toYaml .Values.metrics.serviceMonitor.relabellings | nindent 6 }}
+      {{- if .Values.metrics.serviceMonitor.relabelings }}
+      relabelings: {{- toYaml .Values.metrics.serviceMonitor.relabelings | nindent 6 }}
+      {{- end }}
+      {{- if .Values.metrics.serviceMonitor.metricRelabelings }}
+      metricRelabelings: {{- toYaml .Values.metrics.serviceMonitor.metricRelabelings | nindent 6 }}
```

Comment:
We can take advantage of this `relabellings` typo to avoid a new major. My proposal is to both the old `relabellings` value and the new `relabelings` and `metricRelabelings`.

If `relabellings` is set, the behavior will be the same as the current behavior: `metricRelabelings` will be set to `relabellings`. If, on the other hand, `relabellings` is not set, we can use the new values and set `relabelings` and `metricRelabelings` using the new values.

The description of the old `relabellings` value should be updated to say that the setting is deprecated and will be removed in the next major. 

Do you like this approach?

Reply:
I like this approach really much, I have used your suggestion and refactored it into a minor release.